src/cf-definitions-builder.ts

```diff
@@ -36,14 +36,17 @@ export default class CFDefinitionsBuilder {
     }
   }
 
-  public appendType = (model: CFContentType): CFDefinitionsBuilder => {
+  public appendType = (
+    model: CFContentType,
+    editorInterfaces?: CFEditorInterface[],
```

Comment:
Since we pass in already a single content type, we should also pass in a single editor interface object.